easybuild/easyconfigs/s/SCENICplus/SCENICplus-1.0a1-20240513-foss-2023a.eb

```
@@ -0,0 +1,279 @@
+    }),
+    ('pandoc', '2.3', {
+        'checksums': ['e772c2c6d871146894579828dbaf1efd538eb64fc7e71d4a6b3a11a18baef90d'],
+    }),
+    ('nbsphinx', '0.9.4', {
+        'checksums': ['042a60806fc23d519bc5bef59d95570713913fe442fda759d53e3aaf62104794'],
+    }),
+    ('nbsphinx-link', '1.3.0', {
+        'checksums': ['fa3079a74c0dff1b2079e79a34babe770706ba8aa9cc0609c6dbfd593461a077'],
+    }),
+    ('numpydoc', '1.7.0', {
+        'checksums': ['866e5ae5b6509dcf873fc6381120f5c31acf13b135636c1a81d68c166a95f921'],
+    }),
+    ('pyOpenSSL', '23.2.0', {
+        'modulename': 'OpenSSL',
+        'checksums': ['276f931f55a452e7dea69c7173e984eb2a4407ce413c918aa34b55f82f9b8bac'],
+    }),
+    ('entrypoints', '0.4', {
+        'checksums': ['b706eddaa9218a19ebcd67b56818f05bb27589b1ca9e8d797b74affad4ccacd4'],
+    }),
+    ('ansicolors', '1.1.8', {
+        'modulename': 'colors',
+        'source_tmpl': '%(name)s-%(version)s-py2.py3-none-any.whl',
+        'checksums': ['00d2dde5a675579325902536738dd27e4fac1fd68f773fe36c21044eb559e187'],
+    }),
+    ('papermill', '2.6.0', {
+        'checksums': ['9fe2a91912fd578f391b4cc8d6d105e73124dcd0cde2a43c3c4a1c77ac88ea24'],
+    }),
+    ('mypy_extensions', '1.0.0', {
+        'checksums': ['75dbf8955dc00442a438fc4d0666508a9a97b6bd41aa2f0ffe9d2f2725af0782'],
+    }),
+    ('typing_inspect', '0.9.0', {
+        'checksums': ['b23fc42ff6f6ef6954e4852c1fb512cdd18dbea03134f91f856a95ccc9461f78'],
+    }),
+    ('marshmallow', '3.21.1', {
+        'checksums': ['4e65e9e0d80fc9e609574b9983cf32579f305c718afb30d7233ab818571768c3'],
+    }),
+    ('dataclasses_json', '0.6.4', {
+        'checksums': ['73696ebf24936560cca79a2430cbc4f3dd23ac7bf46ed17f38e5e5e7657a6377'],
+    }),
+    ('loomxpy', '0.4.2', {
+        'preinstallopts': local_loomxpy_preinstallopts,
+        'checksums': ['188411b77e04fa8458c0a7f02cfb3f15b58410a020f81f522957e922e79cdd82'],
+    }),
+    ('pycisTopic', '2.0a-dca4bb6', {
+        'modulename': 'pycisTopic',
+        'preinstallopts': local_pycistopic_preinstallopts,
+        # download commit dca4bb6 to be compatible with pycistarget
+        'source_urls': ['https://github.com/aertslab/pycisTopic/archive/'],
+        'sources': [{'download_filename': 'dca4bb6.tar.gz', 'filename': '%(name)s-%(version)s-dca4bb6.tar.gz'}],
+        'checksums': ['82edc9313bbb587aeb1540fd5b095b6eafb422fa12a5fc97ba090b99b684b240'],
+    }),
+    ('pycistarget', '1.0a1-16d14b9', {
+        'preinstallopts': local_pycistarget_preinstallopts,
+        # download commit 16d14b9 to make '$ scenicplus' work
+        'source_urls': ['https://github.com/aertslab/pycistarget/archive/'],
+        'sources': [{'download_filename': '16d14b9.tar.gz', 'filename': '%(name)s-%(version)s-16d14b9.tar.gz'}],
+        'checksums': ['7285ac03be9d148437442a726366d7c6f9f5f86bc6636369149a2bc2cd8e0e29'],
+    }),
+    ('%(namelower)s', version, {
```

Avoid using `%(namelower)s` in extension name (there's an unresolved bug in EasyBuild framework that leads to not resolving this template):
```suggestion
    ('scenisplus', version, {